app/views/layouts/mountain_view.html.erb

```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
   <div class="mv-header">
-    <div class="mv-header__logo">STYLEGUIDE</div>
+    <div class="mv-header__logo"><a href="/mountain_view">STYLEGUIDE</a></div>
```

Comment:
I believe I have to fix this.  There's a setting to change what the mountain_view location, isn't there?

Reply:
Yeah, this can be set when mounting the engine: https://github.com/devnacho/mountain_view#setting-up-the-style-guide

Not sure how best to get that from the view. If it's too complicated we can leave it as it was before for now.

Reply:
What if we required an alias when mounted? Then we'd know where it is:
```mount MountainView::Engine => "/mountain_view", as: 'mv_style_guide'```

And the link would be ```<%= link_to "STYLEGUIDE", mv_style_guide.root_url %>```

I searched for a  while to see if I could find a way to know whether `mv_style_guide.root_url` existed in routes, but the answers I found seemed to be deprecated.

Reply:
Agree 100% that having a logo that links back to the root is a practice we should follow.

The `rescue` solution looks a bit hacky. I'm not a big fan of inline `rescue`s as they can hide other issues.

Can we check for `responds_to?(:mv_style_guide)` or something like that?